packages/block-editor/src/components/link-control/index.js

```diff
@@ -202,9 +205,9 @@ function LinkControl( {
 										className="block-editor-link-control__search-item-title"
 										href={ value.url }
 									>
-										{ value.title }
+										{ value.title || displayURL }
```

Comment:
I don't think we have a "title" for the link in the case of the format link. So LinkControl should handle properly the case where there's no title.

Reply:
Good catch. Thank you.

Reply:
> I don't think we have a "title" for the link in the case of the format link. So LinkControl should handle properly the case where there's no title.

I pulled this revision out as a separate pull request: #19739